src/MappingOperation/Implementations/MapToMultiple.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace AutoMapperPlus\MappingOperation\Implementations;
+
+use AutoMapperPlus\Exception\UnregisteredMappingException;
+use AutoMapperPlus\MappingOperation\ContextAwareOperation;
+use AutoMapperPlus\MappingOperation\ContextAwareTrait;
+use AutoMapperPlus\MappingOperation\DefaultMappingOperation;
+use AutoMapperPlus\MappingOperation\MapperAwareOperation;
+use AutoMapperPlus\MappingOperation\MapperAwareTrait;
+use Traversable;
+use function is_array;
+
+/**
+ * Class MapTo.
+ * Allows a property to be mapped itself.
+ * @package AutoMapperPlus\MappingOperation\Implementations
+ */
+class MapToMultiple extends DefaultMappingOperation implements
+    MapperAwareOperation,
+    ContextAwareOperation
+{
+    use MapperAwareTrait;
+    use ContextAwareTrait;
+
+    /**
+     * @var string
+     */
+    private $destinationClassList;
+
+    /**
+     * @var array
+     */
+    private $ownContext = [];
+
+    /**
+     * MapTo constructor.
+     * @param string[] $destinationClassList
+     * @param array $context
+     *   Indicates whether or not an array as source value should be treated as
+     *   a collection of elements, or as an array representing an object.
+     * @param array
+     *   $context Optional context that will be merged with the parent's
+     *   context.
+     */
+    public function __construct(
+        array $destinationClassList,
+        array $context = []
```

Comment:
Is there a special use case for this separate context?

Reply:
I'm not using anything from it. I just added it so it is the same as on your other methods. Also I'm passing it to `map` method which I'm using `$returnValue = $this->mapper->map($value, $destinationClass, $context);`